content/en/docs/reference/command-line-tools-reference/feature-gates/VolumeCapacityPriority.md

```diff
@@ -1,14 +0,0 @@
----
-title: VolumeCapacityPriority
-content_type: feature_gate
-_build:
-  list: never
-  render: false
-
-stages:
-  - stage: alpha 
-    defaultValue: false
-    fromVersion: "1.21"
----
-Enable support for prioritizing nodes in different
```

Comment:
Why delete this ?
No feature gate file is supposed to be deleted.

Reply:
Sorry. It was my mistake. I reverted this file.

Reply:
The `toVersion` is inclusive. If the gate is gone in 1.33, that means the alpha stage ends in "1.32", not "1.33".

Reply:
Thank you. I fixed it.